src/System.Net.HttpListener/tests/HttpListenerRequestTests.cs

```diff
@@ -195,6 +194,12 @@ public static IEnumerable<object[]> ContentEncoding_TestData()
         [InlineData("Unknown-Header: Test", false)]
         public async Task IsWebSocketRequest_GetProperty_ReturnsExpected(string webSocketString, bool expected)
         {
+            // Skip on Windows 7 or UAP platforms. WebSocket support is not present.
+            if (PlatformDetection.IsWindows7 || !PlatformDetection.IsNotOneCoreUAP)
```

Comment:
Don't we have a property somewhere that we could use from a ConditionalTheory?  I thought we had one we used in the WebSockets.Client tests?